rpcs3/Emu/Cell/PPUTranslator.cpp

```diff
@@ -231,6 +233,25 @@ Value* PPUTranslator::VecHandleNan(Value* val)
 	return val;
 }
 
+Value* PPUTranslator::VecHandleDenormal(Value* val)
+{
+	const auto type = val->getType();
+	const auto value = type == GetType<u32[4]>() ? val : m_ir->CreateBitCast(val, GetType<u32[4]>());
+
+	const auto mask = SExt(m_ir->CreateICmpEQ(m_ir->CreateAnd(value, Broadcast(RegLoad(m_jm_mask), 4)), ConstantVector::getSplat(4, m_ir->getInt32(0))), GetType<s32[4]>());
+	const auto nz = m_ir->CreateLShr(mask, 9);
+	const auto result = m_ir->CreateAnd(m_ir->CreateNot(nz), value);
+
+	return type == GetType<u32[4]>() ? result : m_ir->CreateBitCast(result, type);
+}
+
+Value* PPUTranslator::VecHandleResult(Value* val)
+{
+	val = g_cfg.core.llvm_ppu_accurate_vector_nan ? VecHandleResult(val) : val;
```

Comment:
I think this will recursively loop forever with _accurate_ setting enabled.
not a pro in this kind of questions.

Reply:
It won't.

Reply:
ok, sorry)

Reply:
Nvm, I see the typo now 🤦

Reply:
Haha, I was right! You are doing incredible job. Wish you luck.